### PR TITLE
18273+18558 Update Extension Request component UI

### PR DESCRIPTION
**Pull request by leodube-aot**
*Issue #:* /bcgov/entity#18273 and /bcgov/entity#18558

*Description of changes:*
- Addresses UI comments from Yui (see tickets listed above for specifics)
- Fixed extension expiry minimum range
- Adjusted field labels and fields column width
- Adjusted spacing between radio menu options
- Adjusted AGM year field validation message
- Adjusted date picker validation triggering event. I will be implementing proper error messages in 18024

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the business-filings-ui license (Apache 2.0).

### CODE REVIEW

File: src/components/AgmExtension/ExtensionRequest.vue [hunk 1]
```diff
@@ -80,18 +81,18 @@
             <v-col
               cols="12"
               sm="3"
+              class="pr-4"
             >
-              <strong>Previous AGM date or a reference date</strong>
```
Comment (leodube-aot) [inline src/components/AgmExtension/ExtensionRequest.vue:84]: Reflects newest UXPin changes

File: src/components/AgmExtension/ExtensionRequest.vue [hunk 2]
```diff
@@ -80,18 +81,18 @@
...
             <v-col
               cols="12"
               sm="9"
             >
               <DatePicker
-                class="pt-2 pl-4"
+                class="pt-2"
                 title="Previous AGM date or a reference date"
                 nudge-right="40"
-                :inputRules="dateRules"
```
Comment (leodube-aot) [inline src/components/AgmExtension/ExtensionRequest.vue:94]: The input rules was causing validation on blur event for the date text field, which could occur when the user clicks on the page while the date picker is still open. I will be implementing validation using the `errorMsg` prop in the wrap-up ticket.

Screenshot 2023-11-24 at 1 05 58 PM

**Comment by bcregistry-sre** [comment]
Temporary Url for review: https://business-filings-dev--pr-592-xzlfntdt.web.app

**Comment by leodube-aot** [comment]
UI fixes approved by Yui, tickets will move to Ready for QA after merge.

File: src/components/AgmExtension/ExtensionRequest.vue (hunk 2 above)
Comment (severinbeauvais) [reply src/components/AgmExtension/ExtensionRequest.vue:94]: Could this component be fixed, instead, to not validate on blur?

**Review by severinbeauvais — APPROVED** [review]

File: src/components/AgmExtension/ExtensionRequest.vue (hunk 2 above)
Comment (leodube-aot) [reply src/components/AgmExtension/ExtensionRequest.vue:94]: Yup I can fix it in the common component

**Pull request merged**